### PR TITLE
CSS & JS for Bok 1 Any% exp table

### PR DESCRIPTION
**Pull request by xDrHellx**
## Files
- Added CSS file containing classes for using Boktai element colors on texts
- Added CSS & JS files related to the Any% exp table

## JavaScript
- Added a JS object used for handling the Any% exp table
- Added a function in said object for adding text-color classes on the corresponding columns of each row to make things easier to read and keep track of

### CODE REVIEW

File: theme/js/any-percent-exp-table.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+                // For each row, get the columns
+                let columns = rows[r].getElementsByTagName("td");
+                for(let c = 0; c < columns.length; c++) {
+
+                    // Add classes to specific columns
+                    switch(c) {
+                        case 1:         // Sol lens EXP
+                        case 2:
+                            columns[c].classList.add("c-sol");
```
Comment (raphaelr) [inline theme/js/any-percent-exp-table.js]: Can this be done with css and the `:nth-child` selector instead? Using .css for styling is preferable to using .js

File: theme/css/colors.css [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/* Elements-related colors */
+.c-sol {
```
Comment (raphaelr) [inline theme/css/colors.css]: Do these colors look good with both the default and the dark mdbook themes?

Comment (xDrHellx) [reply theme/css/colors.css]: Is this good enough?
I've adjusted the .c-dark color, it was too dark to be read properly on dark themes.
elem_colors

File: theme/js/any-percent-exp-table.js (hunk 1 above)
Comment (xDrHellx) [reply theme/js/any-percent-exp-table.js]: I've switched to CSS to do this, see https://github.com/shenef/Boktai-Speedrunning/pull/1/commits/1f28364ff1b17e0e490083c52491f139cf9ac30c

**Pull request merged**